synapse/models/infotech.py

```diff
@@ -1255,6 +1259,45 @@ def getModelDefs(self):
                         'doc': 'An array of ATT&CK technique IDs addressed by the mitigation.',
                     }),
                 )),
+                ('it:mitre:attack:campaign', {}, (
+                    ('name', ('ou:name', {}), {
```

Comment:
Should this instead be ou:campname (so similar to ou:campaign:name?)

Reply:
Great suggestion! Fixed.